tests/providers/amazon/aws/operators/test_ecs.py

```diff
@@ -228,11 +229,11 @@ def test_check_success_tasks_raises(self):
         with pytest.raises(Exception) as ctx:
             self.ecs._check_success_task()
 
-        # Ordering of str(dict) is not guaranteed.
-        assert "This task is not in success state " in str(ctx.value)
-        assert "'name': 'foo'" in str(ctx.value)
-        assert "'lastStatus': 'STOPPED'" in str(ctx.value)
-        assert "'exitCode': 1" in str(ctx.value)
+        expected_last_logs = "\n".join(mock_last_log_messages.return_value)
+        assert (
+            str(ctx.value)
+            == f"This task is not in success state - last logs from Cloudwatch:\n{expected_last_logs}"
+        )
```

Comment:
```suggestion
        assert str(ctx.value) == (
            f"This task is not in success state - last logs from Cloudwatch:\n1\n2\n3\n4\n5"
        )
```

Test code is encouraged to duplicate things, so the assertion call is as close to human perception as possible.